src/main/java/uk/gov/register/db/ItemQueryDAO.java

```diff
@@ -24,12 +24,14 @@
     @RegisterMapper(ItemMapper.class)
     Collection<Item> getAllItemsNoPagination();
 
-    @SqlQuery("select sha256hex, content from item where exists(select 1 from entry where item.sha256hex = entry.sha256hex)")
+    // TODO: This query will need to change to remove `distinct` when download-rsf supports multiple items per entry.
+    @SqlQuery("select distinct ei.sha256hex, i.content from item i join entry_item ei on i.sha256hex = ei.sha256hex")
```

Comment:
I think it would be simpler to use the existing query (but using `entry_item` instead of `entry`). We only care about returning each item once and since they are guaranteed to only appear in the `item` table once, you don't have to worry about using `distinct`. It's also a smaller change.

```
select sha256hex, content from item where exists(select 1 from entry_item ei where item.sha256hex = ei.sha256hex);
```

This is actually more performant:

```
explain select sha256hex, content from item where exists(select 1 from entry_item where item.sha256hex = entry_item.sha256hex);
                                QUERY PLAN
---------------------------------------------------------------------------
 Hash Semi Join  (cost=35.50..87.25 rows=1000 width=191)
   Hash Cond: ((item.sha256hex)::text = (entry_item.sha256hex)::text)
   ->  Seq Scan on item  (cost=0.00..38.00 rows=1000 width=191)
   ->  Hash  (cost=23.00..23.00 rows=1000 width=65)
         ->  Seq Scan on entry_item  (cost=0.00..23.00 rows=1000 width=65)
(5 rows)
```

than:

```
explain select distinct ei.sha256hex, i.content from item i join entry_item ei on i.sha256hex = ei.sha256hex;
                                     QUERY PLAN
------------------------------------------------------------------------------------
 HashAggregate  (cost=92.25..102.25 rows=1000 width=191)
   Group Key: ei.sha256hex, i.content
   ->  Hash Join  (cost=35.50..87.25 rows=1000 width=191)
         Hash Cond: ((i.sha256hex)::text = (ei.sha256hex)::text)
         ->  Seq Scan on item i  (cost=0.00..38.00 rows=1000 width=191)
         ->  Hash  (cost=23.00..23.00 rows=1000 width=65)
               ->  Seq Scan on entry_item ei  (cost=0.00..23.00 rows=1000 width=65)
(7 rows)
```